src/plugins/global-settings/index.js

```diff
@@ -35,6 +39,40 @@ addAction( 'stackable.global-settings.toggle-sidebar', 'toggle', () => {
 } )
 
 const GlobalSettings = () => {
+	const isEditingTemplate = useSelect( select => select( 'core/edit-post' ).isEditingTemplate() )
+
+	/**
+	 * Render the global colors and typography in Gutenberg
+	 *
+	 * WordPress 5.8 introduces block templates.
+	 * When editing blocks inside a template window, the editor is mounted inside
+	 * an `iframe` DOMElement. For the styles to work, we need to mount the styles inside
+	 * the iframe document.
+	 *
+	 * @since 2.17.2
+	 */
+	useEffect( () => {
+		const selector = isEditingTemplate ?
+			document.querySelector( 'iframe[title="Editor canvas"]' ).contentWindow.document.body :
```

Comment:
This might fail if the string is translated to other languages. Try the other attribute:

```suggestion
			document.querySelector( 'iframe[name="editor-canvas"]' ).contentWindow.document.body :
```